src/model/src/phot.F

```diff
@@ -1193,127 +1193,135 @@ END SUBROUTINE CONVCLD_PROP_ACM
                   END IF
                END IF ! not USE_ACM_CLOUD and  ACM_CLOUDS > 0
 
-!(Wei Li)
 !------------------------CANOPY PHOTOLYSIS CORRECTION/REDUCTION Section NOAA-ARL-------------------------------------------
 !Conditions to reduce weighted average of photolysis rates (RJ) due to canopy shading (if user-defined=true); P. C. Campbell
-!Following is based on work of ECCC in GEM-MACHv2.1:  Makar et al. (2017)
+!Following is based on work of ECCC in GEM-MACHv2.1:  Makar et al.(2017)
 !Makar, P., Staebler, R., Akingunola, A. et al. The effects of forest canopy shading and turbulence on boundary layer ozone.
 !Nat Commun 8, 15243 (2017). https://doi.org/10.1038/ncomms15243
 
-                        IF ( CANOPY_SHADE ) THEN ! compute canopy shade reduction factor (RJ_CORR)
-!                                WRITE(*,*) 'LAIE      = ', Met_Data%LAIE( COL,ROW ) ,
-!     &                                     'FCH       = ', Met_Data%FCH( COL,ROW ),
-!     &                                     'FRT       = ', Met_Data%FRT( COL,ROW),
-!     &                                     'POPU      = ', Met_Data%POPU( COL,ROW),
-!     &                                     'CLU       = ', Met_Data%CLU( COL,ROW)
+                      IF ( CANOPY_SHADE ) THEN ! compute canopy shade reduction factor (RJ_CORR)
 
-                            !conditions for grid cells that do NOT have
-                            !a continuous forest canopy
+                        DO LEV = 1, NLAYS !loop through model layers
```

Comment:
Canopy effect should be only applied to lowest 1 or 2 layers, not the whole model layers (NLAYS)

Reply:
we discussed today that let use it for now, loop through all the layers.